app/controllers/registrations_controller.rb

```diff
@@ -13,6 +15,7 @@ def create
   def create_from_json
     build_resource(sign_up_params)
     resource_saved = resource.save
+    build_zoo_user(resource)
```

Comment:
Building zoo user here after we've created a local user account fails the condition where the zoo user has a uniq, duplicate email / login condition. Add this to registrations spec under '#create', 'with_valid_attributes':

``` ruby
context "when the user already exists in zooniverse_home" do
  let!(:existing_zoo_user) { create(:zooniverse_user, login: display_name) }
    
  it "should not create the panotpes user" do
    expect { post :create, user: user_attributes }.not_to change{ User.count }
  end
end
```